ArduCopter/mode_follow.cpp

```diff
@@ -32,6 +32,17 @@ void ModeFollow::exit()
 
 void ModeFollow::run()
 {
+#if HAL_MOUNT_ENABLED
+    Location target_loc;
+    Vector3f veh_vel;
+    AP_Follow *f = AP_Follow::get_singleton();
```

Comment:
Txs for this.  It's looking pretty good, We should add a nullptr check of f and s.

Reply:
actually it looks like the rest of ModeFollow::run() uses "g2.follow" so let's use that instead.